### ISSUE TITLE
Dataset Release Notes

### ISSUE THREAD

**Issue by Cavendish518** [Cavendish518/SFPNet#1]
S.MID beta version 1.2 has been released on July 3, 2024. 

[S.MID version 1.3](https://www.semanticindustry.top/download) has been released on Aug 3, 2024.  In S.MID v1.3, we have optimized the 3D semantic segmentation ground truth labels by correcting several minor errors and enhanced the dataset by including annotations for several unlabeled point clouds in S.MID beta version 1.2 that were previously filtered out by the robot for other downstream tasks. 

The baseline provided in the paper are trained and tested based on SMID_beta_v1_2. Since the dataset has been renewed, the training and test results will be slightly different. We provide a fine-tuned model on S.MID v1.3 in the weight library. The rigorous baseline results are provided in the [leaderboard](https://www.semanticindustry.top/tasks). Thanks!

**Comment by Cavendish518** [comment]
[S.MID version 1.4](https://www.semanticindustry.top/download) has been updated. We have provided rough pose files and update the point clouds' intensity.

**Comment by Kang-8846** [comment]
Hi, I found that the sequences/03/labels/directory is empty (no .label files are provided), while sequences 01 and 02 contain complete label files. When will the labels for sequence 03 be publicly released?

**Comment by Cavendish518** [comment]
> Hi, I found that the sequences/03/labels/directory is empty (no .label files are provided), while sequences 01 and 02 contain complete label files. When will the labels for sequence 03 be publicly released?

Hi, similar to SemanticKITTI, 03 is the test set which contains all the test sequences, therefore, it will not be released publicly.
